style.css

```diff
@@ -0,0 +1,110 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap');
+
+/* General Styles */
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+  min-width: 32.7rem;
+}
+
+body {
+  font-family: poppins, Helvetica, Arial, sans-serif;
+  background-color: #1c1a19;
+}
+
+/* General Styles End */
+
+/* Header Styling */
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2rem;
+  background-color: #3c3a39;
+}
+
+.logo {
+  text-decoration: none;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 700;
+  font-size: 1.8rem;
+  line-height: 2.4rem;
+  color: #fff;
+}
+
+.menu-icon {
+  color: #fff;
+}
+
+/* Header Styling End */
+
+/* Showcase Section Styling */
+
+.main {
+  background-color: #fff;
+}
+
+.showcase-section {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  color: #fff;
+  padding-block-start: 12rem;
+  background-image: url(./images/headline-illustration.png);
+  background-position: 100% 90%;
+  background-repeat: no-repeat;
+}
```

Comment:
You seem to have changed your desired template from the one examined by the previous reviewer.

Kindly adjust your text and background so that they don't overlap. The overlap prevents the text from being clearly visible.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185732019-19833365-db74-4724-b7a8-9d31882eb4c2.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185732014-82f6c426-2122-41fa-8fff-9c700d8de29d.png)